### ISSUE TITLE
livereload.js don't work

### ISSUE THREAD

**Issue by Steve3029** [statianzo/webpack-livereload-plugin#44]
My webpack config:
`plugins: [
    new LiveReloadPlugin({port: 3001})
]`

In html page:
`<script src="http://localhost:3001/livereload.js"></script>`

I have got the following error on Chrome:
"GET http://localhost:3001/livereload.js net::ERR_CONNECTION_REFUSED"

**Comment by statianzo** [comment]
Does your webpack log output `Live reload listening on port 3001`?

**Comment by Steve3029** [comment]
No, I haven't got any log showing live reload listening on port 3001.

**Comment by statianzo** [comment]
If there's no logging, then the plugin server isn't starting. Are you running webpack in watch mode?

What version of webpack are you using? What version of webpack-livereload-plugin? Can you share an entire webpack config that reproduces this issue?

**Comment by Steve3029** [comment]
@statianzo I used  webpack-livereload-plugin v2.1.1 and webpack 4. And the issue has been fixed when I went back to webpack v3.11.0 and webpack-livereload-plugin v1.0.0.

**Comment by statianzo** [comment]
Thanks for the report @Steve3029. I'm reopening this as a reminder to look at v2.1.1 with webpack 4.

**Comment by amiceli** [comment]
Currently with version 2.1.1 and webpack 4.19.1 it works for me.

**Comment by rachelbriggs** [comment]
Same error using webpack 4.29.1 and plugin version 2.2.0

**Comment by Ragnoroct** [comment]
Dumb mistake on my part was running webpack on docker without opening the port for the webpack-livereload-plugin server.

**Comment by andreyvolokitin** [comment]
Same with webpack 5

**Comment by crisamdegracia** [comment]
Took me many days to make webpack-dev-server work in WordPress coz it needs the HTML file, but I can't.
Finally this plugin works!

```
"webpack-livereload-plugin": "^3.0.1",
"webpack": "^5.33.2",
"webpack-cli": "^4.6.0",  

```
`new LiveReloadPlugin({
      protocol: "http",
      hostname : "understraptest.local",
      appendScriptTag: true,
    })`
